clang/lib/Driver/Driver.cpp

```diff
@@ -5433,8 +5433,11 @@ class OffloadingActionBuilder final {
               if (auto *OWA = dyn_cast<OffloadWrapperJobAction>(DeviceAction))
                 OWA->setOffloadKind(Action::OFK_Host);
               Action *CompiledDeviceAction =
-                  C.MakeAction<OffloadWrapperJobAction>(WrapperItems,
-                                                        types::TY_Object);
+                  C.MakeAction<OffloadWrapperJobAction>(FPGAAOTAction,
+                                                        types::TY_Tempfilelist);
+              if (auto *OWA =
+                      dyn_cast<OffloadWrapperJobAction>(CompiledDeviceAction))
+                OWA->setIndividualWrap(true);
```

Comment:
Can you explain this code?
Here `FPGAAOTAction` is passed and down below `FPGAArchiveWrapperInputs`. And for both cases we set `IndividualWrap` to True.
Maybe a comment would be helpful.